ibis/tests/all/conftest.py

```diff
@@ -366,3 +371,24 @@ def get_common_spark_testing_client(data_directory, connect):
     df_udf_random.createOrReplaceTempView('udf_random')
 
     return _spark_testing_client
+
+
+@pytest.fixture
```

Comment:
suprised we don't already have something like this, can you check (and if not, ok, follows to use this fixture would be good), can you create an issue

Reply:
thanks @jreback . done: https://github.com/ibis-project/ibis/issues/2148